From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sun, 8 Jul 2018 13:12:04 +0200
Subject: fix misspelled 'allows to' -> 'allows one to'

Fixup small grammar issues around used 'allows to ...' almost in
code comments.
---
 3d-viewer/3d_viewer/eda_3d_viewer.h         |  2 +-
 common/gal/opengl/gl_builtin_shaders.cpp    | 24 ++++++++++++------------
 eeschema/sch_edit_frame.h                   |  6 +++---
 eeschema/sch_sheet_path.h                   |  2 +-
 eeschema/sim/netlist_exporter_pspice_sim.h  |  2 +-
 gerbview/tools/selection_tool.h             | 12 +++++++-----
 include/basic_gal.h                         |  2 +-
 include/plotter.h                           | 12 ++++++------
 include/tool/action_manager.h               |  4 ++--
 include/tool/context_menu.h                 |  4 ++--
 include/tool/tool_event.h                   |  2 +-
 include/view/view.h                         | 12 ++++++------
 pcbnew/connectivity_data.h                  |  4 ++--
 pcbnew/dialogs/dialog_pns_settings_base.fbp |  2 +-
 pcbnew/netinfo.h                            |  2 +-
 pcbnew/tools/drawing_tool.h                 |  4 ++--
 pcbnew/tools/edit_points.h                  |  6 +++---
 pcbnew/tools/point_editor.cpp               |  4 ++--
 pcbnew/tools/selection_tool.h               |  9 +++++----
 19 files changed, 59 insertions(+), 56 deletions(-)

diff --git a/3d-viewer/3d_viewer/eda_3d_viewer.h b/3d-viewer/3d_viewer/eda_3d_viewer.h
index 1f4e37a..25568a5 100644
--- a/3d-viewer/3d_viewer/eda_3d_viewer.h
+++ b/3d-viewer/3d_viewer/eda_3d_viewer.h
@@ -65,7 +65,7 @@ class EDA_3D_VIEWER : public KIWAY_PLAYER
     /**
      * Request reloading the 3D view. However the request will be executed
      * only when the 3D canvas is refreshed.
-     * It allows to prepare changes and request for 3D rebuild only when all
+     * It allows one to prepare changes and request for 3D rebuild only when all
      * changes are committed.
      * This is made because the 3D rebuild can take a long time, and this rebuild
      * cannot always made after each change, for calculation time reason.
diff --git a/common/gal/opengl/gl_builtin_shaders.cpp b/common/gal/opengl/gl_builtin_shaders.cpp
index 60f2962..4da38e4 100644
--- a/common/gal/opengl/gl_builtin_shaders.cpp
+++ b/common/gal/opengl/gl_builtin_shaders.cpp
@@ -519,7 +519,7 @@ const char smaa_base_shader_p1[] = R"SHADER_SOURCE(
  *      - DX10.1:   D3D10_STANDARD_MULTISAMPLE_PATTERN or
  *      - DX11:     D3D11_STANDARD_MULTISAMPLE_PATTERN
  *
- *    This allows to ensure that the subsample order matches the table in
+ *    This allows one to ensure that the subsample order matches the table in
  *    @SUBSAMPLE_INDICES.
  *
  *    (*) In the case of DX10, we refer the reader to:
@@ -620,8 +620,8 @@ const char smaa_base_shader_p1[] = R"SHADER_SOURCE(
  * performance.
  *
  * Range: [0, 0.5]
- *   0.1 is a reasonable value, and allows to catch most visible edges.
- *   0.05 is a rather overkill value, that allows to catch 'em all.
+ *   0.1 is a reasonable value, and allows one to catch most visible edges.
+ *   0.05 is a rather overkill value, that allows one to catch 'em all.
  *
  *   If temporal supersampling is used, 0.2 could be a reasonable value, as low
  *   contrast edges are properly filtered by just 2x.
@@ -686,7 +686,7 @@ const char smaa_base_shader_p2[] = R"SHADER_SOURCE(
  * If there is an neighbor edge that has SMAA_LOCAL_CONTRAST_FACTOR times
  * bigger contrast than current edge, current edge will be discarded.
  *
- * This allows to eliminate spurious crossing edges, and is based on the fact
+ * This allows one to eliminate spurious crossing edges, and is based on the fact
  * that, if there is too much contrast in a direction, that will hide
  * perceptually contrast in the other neighbors.
  */
@@ -695,7 +695,7 @@ const char smaa_base_shader_p2[] = R"SHADER_SOURCE(
 #endif
 
 /**
- * Predicated thresholding allows to better preserve texture details and to
+ * Predicated thresholding allows one to better preserve texture details and to
  * improve performance, by decreasing the number of detected edges using an
  * additional buffer like the light accumulation buffer, object ids or even the
  * depth buffer (the depth buffer usage may be limited to indoor or short range
@@ -742,7 +742,7 @@ const char smaa_base_shader_p2[] = R"SHADER_SOURCE(
 #endif
 
 /**
- * Temporal reprojection allows to remove ghosting artifacts when using
+ * Temporal reprojection allows one to remove ghosting artifacts when using
  * temporal supersampling. We use the CryEngine 3 method which also introduces
  * velocity weighting. This feature is of extreme importance for totally
  * removing ghosting. More information here:
@@ -757,8 +757,8 @@ const char smaa_base_shader_p2[] = R"SHADER_SOURCE(
 #endif
 
 /**
- * SMAA_REPROJECTION_WEIGHT_SCALE controls the velocity weighting. It allows to
- * remove ghosting trails behind the moving object, which are not removed by
+ * SMAA_REPROJECTION_WEIGHT_SCALE controls the velocity weighting. It allows one
+ * to remove ghosting trails behind the moving object, which are not removed by
  * just using reprojection. Using low values will exhibit ghosting, while using
  * high values will disable temporal supersampling under motion.
  *
@@ -1158,7 +1158,7 @@ float4 SMAADecodeDiagBilinearAccess(float4 e) {
 }
 
 /**
- * These functions allows to perform diagonal pattern searches.
+ * These functions allows one to perform diagonal pattern searches.
  */
 float2 SMAASearchDiag1(SMAATexture2D(edgesTex), float2 texcoord, float2 dir, out float2 e) {
     float4 coord = float4(texcoord, -1.0, 1.0);
@@ -1290,7 +1290,7 @@ float2 SMAACalculateDiagWeights(SMAATexture2D(edgesTex), SMAATexture2D(areaTex),
 // Horizontal/Vertical Search Functions
 
 /**
- * This allows to determine how much length should we add in the last step
+ * This allows one to determine how much length should we add in the last step
  * of the searches. It takes the bilinearly interpolated edge (see
  * @PSEUDO_GATHER4), and adds 0, 1 or 2, depending on which edges and
  * crossing edges are active.
@@ -1322,8 +1322,8 @@ float SMAASearchXLeft(SMAATexture2D(edgesTex), SMAATexture2D(searchTex), float2
      * @PSEUDO_GATHER4
      * This texcoord has been offset by (-0.25, -0.125) in the vertex shader to
      * sample between edge, thus fetching four edges in a row.
-     * Sampling with different offsets in each direction allows to disambiguate
-     * which edges are active from the four fetched ones.
+     * Sampling with different offsets in each direction allows one to
+     * disambiguate which edges are active from the four fetched ones.
      */
     float2 e = float2(0.0, 1.0);
     while (texcoord.x > end &&
diff --git a/eeschema/sch_edit_frame.h b/eeschema/sch_edit_frame.h
index be34627..fc130c6 100644
--- a/eeschema/sch_edit_frame.h
+++ b/eeschema/sch_edit_frame.h
@@ -898,9 +898,9 @@ private:
     void OnRescueProject( wxCommandEvent& event );
     void OnRemapSymbols( wxCommandEvent& aEvent );
 
-    // a helper function to run the dialog that allows to rename the symbol library Id of
-    // groups of components, for instance after a symbol has moved from a library to
-    // another library
+    // a helper function to run the dialog that allows one to rename the symbol
+    // library Id of groups of components, for instance after a symbol has moved
+    // from a library to another library
     void OnEditComponentSymbolsId( wxCommandEvent& aEvent );
     void OnPreferencesOptions( wxCommandEvent& event );
     void OnCancelCurrentCommand( wxCommandEvent& aEvent );
diff --git a/eeschema/sch_sheet_path.h b/eeschema/sch_sheet_path.h
index a8f708e..9ad30ce 100644
--- a/eeschema/sch_sheet_path.h
+++ b/eeschema/sch_sheet_path.h
@@ -71,7 +71,7 @@
  *      corresponding references and sheet path
  *
  * The class SCH_SHEET_PATH handles paths used to access a sheet.  The class
- * SCH_SHEET_LIST allows to handle the full (or partial) list of sheets and
+ * SCH_SHEET_LIST allows one to handle the full (or partial) list of sheets and
  * their paths in a complex hierarchy.  The class EDA_ScreenList allows one
  * to handle the list of SCH_SCREEN. It is useful to clear or save data,
  * but is not suitable to handle the full complex hierarchy possibilities
diff --git a/eeschema/sim/netlist_exporter_pspice_sim.h b/eeschema/sim/netlist_exporter_pspice_sim.h
index f3f09ca..31093dd 100644
--- a/eeschema/sim/netlist_exporter_pspice_sim.h
+++ b/eeschema/sim/netlist_exporter_pspice_sim.h
@@ -32,7 +32,7 @@
 
 #include "sim_types.h"
 
-/// Special netlist exporter flavor that allows to override simulation commands
+/// Special netlist exporter flavor that allows one to override simulation commands
 class NETLIST_EXPORTER_PSPICE_SIM : public NETLIST_EXPORTER_PSPICE
 {
 public:
diff --git a/gerbview/tools/selection_tool.h b/gerbview/tools/selection_tool.h
index 06a693a..4b42e4c 100644
--- a/gerbview/tools/selection_tool.h
+++ b/gerbview/tools/selection_tool.h
@@ -124,8 +124,9 @@ public:
 private:
     /**
      * Function selectPoint()
-     * Selects an item pointed by the parameter aWhere. If there is more than one item at that
-     * place, there is a menu displayed that allows to choose the item.
+     * Selects an item pointed by the parameter aWhere. If there is more than
+     * one item at that place, there is a menu displayed that allows one to
+     * choose the item.
      *
      * @param aWhere is the place where the item should be selected.
      * @param aAllowDisambiguation decides what to do in case of disambiguation. If true, then
@@ -145,7 +146,8 @@ private:
 
     /**
      * Function selectMultiple()
-     * Handles drawing a selection box that allows to select many items at the same time.
+     * Handles drawing a selection box that allows one to select many items at
+     * the same time.
      *
      * @return true if the function was cancelled (i.e. CancelEvent was received).
      */
@@ -159,8 +161,8 @@ private:
 
     /**
      * Function disambiguationMenu()
-     * Handles the menu that allows to select one of many items in case there is more than one
-     * item at the selected point (@see selectCursor()).
+     * Handles the menu that allows one to select one of many items in case
+     * there is more than one item at the selected point (@see selectCursor()).
      *
      * @param aItems contains list of items that are displayed to the user.
      */
diff --git a/include/basic_gal.h b/include/basic_gal.h
index bbada69..c6fda68 100644
--- a/include/basic_gal.h
+++ b/include/basic_gal.h
@@ -38,7 +38,7 @@ class PLOTTER;
  * class BASIC_GAL is a minimal GAL implementation to draw, plot and convert
  * stroke texts to a set of segments for DRC tests, and to calculate text sizes.
  *
- * Currently it allows to use GAL and STROKE_FONT methods in legacy draw mode
+ * Currently it allows one to use GAL and STROKE_FONT methods in legacy draw mode
  * (using wxDC functions) in plot functions only for texts.
  * It is used also to calculate the text bounding boxes
  *
diff --git a/include/plotter.h b/include/plotter.h
index 3273088..7021a04 100644
--- a/include/plotter.h
+++ b/include/plotter.h
@@ -435,7 +435,7 @@ public:
     }
 
     /**
-     * calling this function allows to define the beginning of a group
+     * calling this function allows one to define the beginning of a group
      * of drawing items, for instance in SVG  or Gerber format.
      * (example: group all segments of a letter or a text)
      * @param aData can define any parameter
@@ -444,8 +444,8 @@ public:
     virtual void StartBlock( void* aData ) {}
 
     /**
-     * calling this function allows to define the end of a group of drawing items
-     * for instance in SVG  or Gerber format.
+     * calling this function allows one to define the end of a group of drawing
+     * items for instance in SVG  or Gerber format.
      * the group is started by StartBlock()
      * @param aData can define any parameter
      * for most of plotters: do nothing
@@ -1137,15 +1137,15 @@ public:
     void UseX2NetAttributes( bool aEnable ) { m_useNetAttributes = aEnable; }
 
     /**
-     * calling this function allows to define the beginning of a group
+     * calling this function allows one to define the beginning of a group
      * of drawing items (used in X2 format with netlist attributes)
      * @param aData can define any parameter
      */
     virtual void StartBlock( void* aData ) override;
 
     /**
-     * calling this function allows to define the end of a group of drawing items
-     * the group is started by StartBlock()
+     * calling this function allows one to define the end of a group of drawing
+     * items the group is started by StartBlock()
      * (used in X2 format with netlist attributes)
      * @param aData can define any parameter
      */
diff --git a/include/tool/action_manager.h b/include/tool/action_manager.h
index 34809e0..63649b8 100644
--- a/include/tool/action_manager.h
+++ b/include/tool/action_manager.h
@@ -36,8 +36,8 @@ class TOOL_ACTION;
 /**
  * Class ACTION_MANAGER
  *
- * Takes care of TOOL_ACTION objects. Registers them and allows to run them using associated
- * hot keys, names or ids.
+ * Takes care of TOOL_ACTION objects. Registers them and allows one to run them
+ * using associated hot keys, names or ids.
  */
 class ACTION_MANAGER
 {
diff --git a/include/tool/context_menu.h b/include/tool/context_menu.h
index cdbc335..8f8e7b7 100644
--- a/include/tool/context_menu.h
+++ b/include/tool/context_menu.h
@@ -96,8 +96,8 @@ public:
      * Adds a context menu as a submenu. The difference between this function and wxMenu::AppendSubMenu()
      * is the capability to handle icons.
      * @param aMenu is the submenu to be added.
-     * @param aExpand allows to add all entries from the menu as individual entries rather than
-     *                add everything as a submenu.
+     * @param aExpand allows one to add all entries from the menu as individual
+     *                entries rather than add everything as a submenu.
      */
     std::list<wxMenuItem*> Add( CONTEXT_MENU* aMenu, bool aExpand = false );
 
diff --git a/include/tool/tool_event.h b/include/tool/tool_event.h
index adbd0d0..f176db5 100644
--- a/include/tool/tool_event.h
+++ b/include/tool/tool_event.h
@@ -100,7 +100,7 @@ enum TOOL_ACTIONS
     // This event is sent *after* undo/redo command is performed.
     TA_UNDO_REDO_POST       = 0x40000,
 
-    // Tool action (allows to control tools).
+    // Tool action (allows one to control tools).
     TA_ACTION               = 0x80000,
 
     // Tool activation event.
diff --git a/include/view/view.h b/include/view/view.h
index 3938bfd..7a242b3 100644
--- a/include/view/view.h
+++ b/include/view/view.h
@@ -720,8 +720,8 @@ private:
      *
      * @param aItem is the item to be drawn.
      * @param aLayer is the layer which should be drawn.
-     * @param aImmediate dictates the way of drawing - it allows to force immediate drawing mode
-     * for cached items.
+     * @param aImmediate dictates the way of drawing - it allows one to force
+     * immediate drawing mode for cached items.
      */
     void draw( VIEW_ITEM* aItem, int aLayer, bool aImmediate = false );
 
@@ -730,8 +730,8 @@ private:
      * Draws an item on all layers that the item uses.
      *
      * @param aItem is the item to be drawn.
-     * @param aImmediate dictates the way of drawing - it allows to force immediate drawing mode
-     * for cached items.
+     * @param aImmediate dictates the way of drawing - it allows one to force
+     * immediate drawing mode for cached items.
      */
     void draw( VIEW_ITEM* aItem, bool aImmediate = false );
 
@@ -740,8 +740,8 @@ private:
      * Draws a group of items on all layers that those items use.
      *
      * @param aGroup is the group to be drawn.
-     * @param aImmediate dictates the way of drawing - it allows to force immediate drawing mode
-     * for cached items.
+     * @param aImmediate dictates the way of drawing - it allows one to force
+     * immediate drawing mode for cached items.
      */
     void draw( VIEW_GROUP* aGroup, bool aImmediate = false );
 
diff --git a/pcbnew/connectivity_data.h b/pcbnew/connectivity_data.h
index 3f90f03..bf9c57e 100644
--- a/pcbnew/connectivity_data.h
+++ b/pcbnew/connectivity_data.h
@@ -206,7 +206,7 @@ public:
      * Function GetConnectedItems()
      * Returns a list of items connected to a source item aItem.
      * @param aItem is the reference item to find other connected items.
-     * @param aTypes allows to filter by item types.
+     * @param aTypes allows one to filter by item types.
      */
     const std::vector<BOARD_CONNECTED_ITEM*> GetConnectedItems( const BOARD_CONNECTED_ITEM* aItem,
             const KICAD_T aTypes[] ) const;
@@ -215,7 +215,7 @@ public:
      * Function GetNetItems()
      * Returns the list of items that belong to a certain net.
      * @param aNetCode is the net code.
-     * @param aTypes allows to filter by item types.
+     * @param aTypes allows one to filter by item types.
      */
     const std::vector<BOARD_CONNECTED_ITEM*> GetNetItems( int aNetCode,
             const KICAD_T aTypes[] ) const;
diff --git a/pcbnew/dialogs/dialog_pns_settings_base.fbp b/pcbnew/dialogs/dialog_pns_settings_base.fbp
index 1d72be0..6a0a65f 100644
--- a/pcbnew/dialogs/dialog_pns_settings_base.fbp
+++ b/pcbnew/dialogs/dialog_pns_settings_base.fbp
@@ -968,7 +968,7 @@
                                 <property name="style"></property>
                                 <property name="subclass"></property>
                                 <property name="toolbar_pane">0</property>
-                                <property name="tooltip">(Highlight collisions mode only) - allows to establish a track even if is violating the DRC rules.</property>
+                                <property name="tooltip">(Highlight collisions mode only) - allows one to establish a track even if is violating the DRC rules.</property>
                                 <property name="validator_data_type"></property>
                                 <property name="validator_style">wxFILTER_NONE</property>
                                 <property name="validator_type">wxDefaultValidator</property>
diff --git a/pcbnew/netinfo.h b/pcbnew/netinfo.h
index e379c7e..e3c7de7 100644
--- a/pcbnew/netinfo.h
+++ b/pcbnew/netinfo.h
@@ -294,7 +294,7 @@ public:
     /**
      * Function Translate
      * Translates net number according to the map prepared by Update() function. It
-     * allows to have items stored with consecutive net codes.
+     * allows one to have items stored with consecutive net codes.
      * @param aNetCode is an old net code.
      * @return Net code that follows the mapping.
      */
diff --git a/pcbnew/tools/drawing_tool.h b/pcbnew/tools/drawing_tool.h
index 3160d07..82cea0c 100644
--- a/pcbnew/tools/drawing_tool.h
+++ b/pcbnew/tools/drawing_tool.h
@@ -114,8 +114,8 @@ public:
 
     /**
      * Function PlaceText()
-     * Displays a dialog that allows to input text and its settings and then lets the user decide
-     * where to place the text in editor.
+     * Displays a dialog that allows one to input text and its settings and then
+     * lets the user decide where to place the text in editor.
      */
     int PlaceText( const TOOL_EVENT& aEvent );
 
diff --git a/pcbnew/tools/edit_points.h b/pcbnew/tools/edit_points.h
index f4d925f..ae9aa1b 100644
--- a/pcbnew/tools/edit_points.h
+++ b/pcbnew/tools/edit_points.h
@@ -182,9 +182,9 @@ private:
 /**
  * Class EDIT_LINE
  *
- * Represents a line connecting two EDIT_POINTs. That allows to move them both by dragging the
- * EDIT_POINT in the middle. As it uses references to EDIT_POINTs, all coordinates are
- * automatically synchronized.
+ * Represents a line connecting two EDIT_POINTs. That allows one to move them
+ * both by dragging the EDIT_POINT in the middle. As it uses references to
+ * EDIT_POINTs, all coordinates are automatically synchronized.
  */
 class EDIT_LINE : public EDIT_POINT
 {
diff --git a/pcbnew/tools/point_editor.cpp b/pcbnew/tools/point_editor.cpp
index 39e1d25..a593bc9 100644
--- a/pcbnew/tools/point_editor.cpp
+++ b/pcbnew/tools/point_editor.cpp
@@ -863,8 +863,8 @@ bool POINT_EDITOR::removeCornerCondition( const SELECTION& )
 
     // Check if there are enough vertices so one can be removed without
     // degenerating the polygon.
-    // The first condition allows to remove all corners from holes (when there
-    // are only 2 vertices left, a hole is removed).
+    // The first condition allows one to remove all corners from holes (when
+    // there are only 2 vertices left, a hole is removed).
     if( vertexIdx.m_contour == 0 && polyset.Polygon( vertexIdx.m_polygon )[vertexIdx.m_contour].PointCount() <= 3 )
         return false;
 
diff --git a/pcbnew/tools/selection_tool.h b/pcbnew/tools/selection_tool.h
index 2461d20..54eab30 100644
--- a/pcbnew/tools/selection_tool.h
+++ b/pcbnew/tools/selection_tool.h
@@ -146,7 +146,7 @@ private:
     /**
      * Function selectPoint()
      * Selects an item pointed by the parameter aWhere. If there is more than one item at that
-     * place, there is a menu displayed that allows to choose the item.
+     * place, there is a menu displayed that allows one to choose the item.
      *
      * @param aWhere is the place where the item should be selected.
      * @param aOnDrag indicates whether a drag operation is being performed.
@@ -172,7 +172,8 @@ private:
 
     /**
      * Function selectMultiple()
-     * Handles drawing a selection box that allows to select many items at the same time.
+     * Handles drawing a selection box that allows one to select many items at
+     * the same time.
      *
      * @return true if the function was cancelled (i.e. CancelEvent was received).
      */
@@ -242,8 +243,8 @@ private:
 
     /**
      * Function disambiguationMenu()
-     * Handles the menu that allows to select one of many items in case there is more than one
-     * item at the selected point (@see selectCursor()).
+     * Handles the menu that allows one to select one of many items in case
+     * there is more than one item at the selected point (@see selectCursor()).
      *
      * @param aItems contains list of items that are displayed to the user.
      */
